content/documentation/pages/2-concepts/3-streams.md

```diff
@@ -19,38 +22,38 @@ Deployed stream applications communicate over a messaging middleware product.
 We provide pre-built stream applications that communicate over [RabbitMQ](https://www.rabbitmq.com) or
 [Kafka](https://kafka.apache.org) and that you can use to integrate with various data products.
 
-The following [messaging middleware products](https://cloud.spring.io/spring-cloud-stream/#binder-implementations) are also suppoeted:
+The following messaging middleware products are currently supported by Spring Cloud Data Flow:
```

Comment:
It'd still be nice to keep the link because the list continues to grow on SCSt, and the bullets below may not be up to date always.

Reply:
If you could add the newer ones to the list, that'd help.